### PR TITLE
Tutorial Videos and Removed Glossary Links

### PR DESCRIPTION
**Pull request by baileysingleton**
fixes issue #246 

## Changes in this PR.

- [X] Removed Glossary From Resource page
- [X] Linked tutorial video to go to a page that has videos that can be added by admin
- [X] Ignore changes to requirements.txt, those are changes made in pull request #277 
- [x] Get layout approved/make changes to it if necessary
- [x] Get video size confirmation - how large should the video player be for each video?

## Testing this PR.

1. Launch Website, navigate to Resources
2. Notice the lack of a Glossary Link, click on "Tutorial Videos" to be redirected to tutorial video page
3. If you'd like, log in as admin, upload a video to the tutorial_videos section, and they will appear on the page.

### Expected Output.

```
Tutorial page has been linked from the resources page
Videos added to the tutorial videos page via admin show up
```
@osuosl/devs

### CODE REVIEW

File: streamwebs_frontend/streamwebs/templates/streamwebs/resources/resources_tutorial_videos.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    <tbody>
+      {% for video in data %}
+        <tr>
+          <td>
+            <h4>
+              {{ video.name }}
+            </h4>
+          </td>
+          <td>
+            <video width="320" height="240" controls>
```
Comment (baileysingleton) [inline streamwebs_frontend/streamwebs/templates/streamwebs/resources/resources_tutorial_videos.html]: Not sure what the size of the video player shouid be. I have it set to 320x240, but can be fullscreened

**Comment by baileysingleton** [comment]
@alxngyn My travis build is failing, not sure what the error code means, or how to go about fixing it. I'll probably ask you about it on Monday, the 26th.

**Comment by alxngyn** [comment]
you have a flake8 error
```
The command "flake8 data_scripts" exited with 0.
0.96s$ flake8 streamwebs_frontend/ --exclude streamwebs_frontend/streamwebs/migrations
streamwebs_frontend/streamwebs/views/general.py:972:1: E302 expected 2 blank lines, found 1
The command "flake8 streamwebs_frontend/ --exclude streamwebs_frontend/streamwebs/migrations" exited with 1.
```

**Comment by alxngyn** [comment]
- no urls.py changed, so no routing exists for `http://localhost:8000/resources/tutorial-videos`
- glossary still in resources template

**Comment by JerryPeng0112** [comment]
Need to add validation for file upload.
Also, maybe redirect to tutorial page or resource page after successful video upload, or display a message notifying the user.

File: streamwebs_frontend/streamwebs/views/general.py [hunk 2]
```diff
@@ -993,9 +1006,17 @@ def resources_upload(request):
                     'You have successfully uploaded a new publication.'
                 )
                 return redirect(reverse('streamwebs:resources-publications'))
-            # elif res.res_type == 'tutorial_video':
-                # process the url
-                # redirect to video page
+            elif res.res_type == 'tutorial_video':
+                if hasattr(res, 'downloadale'):
```
Comment (Kennric) [inline streamwebs_frontend/streamwebs/views/general.py]: downloadale? What is going on here exactly?

Comment (baileysingleton) [inline streamwebs_frontend/streamwebs/views/general.py]: Sorry, that was pushed by mistake. Pushing the fixed code right now.

**Comment by JerryPeng0112** [comment]
++ good job, ready to merge

**Pull request merged**